### DISCUSSION TITLE
Ti may be i cannot see in my extension there is no episode found

### DISCUSSION THREAD

**Discussion by Atharva-kastur** [aniyomiorg/aniyomi#1480] [General]
Please somebody tell me how to solve it

**Comment by Domipoke** [comment]
if the problem is with manga extension
Try this:
Go on ...
then settings
then browse
click manga repositofies
add this url:
https://raw.githubusercontent.com/keiyoushi/extensions/repo/index.min.json

**Comment by Atharva-kastur** [comment]
Yes that so Teri fine yo do something like that problem so do better on it and fix this bug
